.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+
+go:
+  - 1.x
```

Comment:
I don't know what this means, but I would list out explicit versions, e.g. 1.8, 1.9 and soon 1.10 (and leave master there too)

Reply:
'x' means latest here. I thought using it would be easier than having to change to the latest stable Go release all the time.

wdyt?

Reply:
I thought master did that already. Personally I would still list 1.8 and 1.9 explicitly, but up to you.

Reply:
master is for the bleeding edge, whereas this is more for latest stable.

I think I'll add 1.8 and 1.9 explicitly, but also keep 1.x so we don't get caught off guard when a new release comes out.

Reply:
Done.